docs/examples.md

```
@@ -583,3 +583,29 @@ Generate `.ics` files with categories, language, and subdivisions.
+
+COUNTRY_CODE: ISO country code (IN, US, etc.)
+YEAR: Single year (2025) or range (2020-2025)
+public: Optional argument to include only public holidays
+
+Description
+The script:
+
+  Dynamically loads country holidays
+  Detects supported categories
+  Generates .ics files for each category
+  Supports filtering public holidays
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)